<!DOCTYPE html>
<html lang="en" data-dark="false">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <!--
  put your analytics (e.g. Google Analytics) tracking code here
-->

  <!--
  put your search engine verification (e.g. Google Search Console) tag here
-->

  


























<meta name="viewport" content="width=device-width, initial-scale=1">

<title>Research | Voight Lab</title>

<link rel="icon" href="/images/icon.png">

<meta name="title" content="Research">
<meta name="description" content="An engaging 1-3 sentence description of your lab.">

<meta property="og:title" content="Research">
<meta property="og:site_title" content="Voight Lab">
<meta property="og:description" content="An engaging 1-3 sentence description of your lab.">
<meta property="og:url" content="">
<meta property="og:image" content="/images/share.jpg">
<meta property="og:locale" content="en_US">

<meta property="twitter:title" content="Research">
<meta property="twitter:description" content="An engaging 1-3 sentence description of your lab.">
<meta property="twitter:url" content="">
<meta property="twitter:card" content="summary_large_image">
<meta property="twitter:image" content="/images/share.jpg">


  <meta property="og:type" content="website">


<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "WebSite",
    
    "name": "Research",
    "description": "An engaging 1-3 sentence description of your lab.",
    "headline": "Research",
    "publisher": {
      "@type": "Organization",
      "logo": { "@type": "ImageObject", "url": "/images/icon.png" }
    },
    "url": ""
  }
</script>

<link rel="alternate" type="application/rss+xml" href="/feed.xml">

  <!-- Google Fonts -->
<!-- automatically get url from fonts used in theme file -->

<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?display=swap&&family=Barlow:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600&amp;family=Roboto+Mono:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600" rel="stylesheet">

<!-- Font Awesome icons (load asynchronously due to size) -->

<link href="https://use.fontawesome.com/releases/v6.3.0/css/all.css" rel="preload" as="style" onload="this.onload = null; this.rel = 'stylesheet';">
<noscript>
  <link href="https://use.fontawesome.com/releases/v6.3.0/css/all.css" rel="stylesheet">
</noscript>

  <!-- third party styles -->
<!-- https://stylishthemes.github.io/Syntax-Themes/pygments/ -->
<link href="https://cdn.jsdelivr.net/gh/StylishThemes/Syntax-Themes/pygments/css-github/pygments-tomorrow-night-eighties.css" rel="stylesheet">

<!-- include all sass in styles folder -->


  
    <link href="/_styles/-theme.css" rel="stylesheet">
  

  
    <link href="/_styles/alert.css" rel="stylesheet">
  

  
    <link href="/_styles/all.css" rel="stylesheet">
  

  
    <link href="/_styles/anchor.css" rel="stylesheet">
  

  
    <link href="/_styles/background.css" rel="stylesheet">
  

  
    <link href="/_styles/body.css" rel="stylesheet">
  

  
    <link href="/_styles/bold.css" rel="stylesheet">
  

  
    <link href="/_styles/button.css" rel="stylesheet">
  

  
    <link href="/_styles/card.css" rel="stylesheet">
  

  
    <link href="/_styles/checkbox.css" rel="stylesheet">
  

  
    <link href="/_styles/citation.css" rel="stylesheet">
  

  
    <link href="/_styles/code.css" rel="stylesheet">
  

  
    <link href="/_styles/cols.css" rel="stylesheet">
  

  
    <link href="/_styles/dark-toggle.css" rel="stylesheet">
  

  
    <link href="/_styles/feature.css" rel="stylesheet">
  

  
    <link href="/_styles/figure.css" rel="stylesheet">
  

  
    <link href="/_styles/float.css" rel="stylesheet">
  

  
    <link href="/_styles/font.css" rel="stylesheet">
  

  
    <link href="/_styles/footer.css" rel="stylesheet">
  

  
    <link href="/_styles/form.css" rel="stylesheet">
  

  
    <link href="/_styles/grid.css" rel="stylesheet">
  

  
    <link href="/_styles/header.css" rel="stylesheet">
  

  
    <link href="/_styles/heading.css" rel="stylesheet">
  

  
    <link href="/_styles/highlight.css" rel="stylesheet">
  

  
    <link href="/_styles/icon.css" rel="stylesheet">
  

  
    <link href="/_styles/image.css" rel="stylesheet">
  

  
    <link href="/_styles/link.css" rel="stylesheet">
  

  
    <link href="/_styles/list.css" rel="stylesheet">
  

  
    <link href="/_styles/main.css" rel="stylesheet">
  

  
    <link href="/_styles/paragraph.css" rel="stylesheet">
  

  
    <link href="/_styles/portrait.css" rel="stylesheet">
  

  
    <link href="/_styles/post-excerpt.css" rel="stylesheet">
  

  
    <link href="/_styles/post-info.css" rel="stylesheet">
  

  
    <link href="/_styles/post-nav.css" rel="stylesheet">
  

  
    <link href="/_styles/quote.css" rel="stylesheet">
  

  
    <link href="/_styles/rule.css" rel="stylesheet">
  

  
    <link href="/_styles/search-box.css" rel="stylesheet">
  

  
    <link href="/_styles/search-info.css" rel="stylesheet">
  

  
    <link href="/_styles/section.css" rel="stylesheet">
  

  
    <link href="/_styles/table.css" rel="stylesheet">
  

  
    <link href="/_styles/tags.css" rel="stylesheet">
  

  
    <link href="/_styles/textbox.css" rel="stylesheet">
  

  
    <link href="/_styles/tooltip.css" rel="stylesheet">
  

  
    <link href="/_styles/util.css" rel="stylesheet">
  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  


<!-- include all css in styles folder -->



  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2" defer></script>
<script src="https://unpkg.com/tippy.js@6" defer></script>
<script src="https://unpkg.com/mark.js@8" defer></script>

<!-- include all js in scripts folder -->


  <script src="/_scripts/anchors.js"></script>

  <script src="/_scripts/dark-mode.js"></script>

  <script src="/_scripts/fetch-tags.js"></script>

  <script src="/_scripts/search.js"></script>

  <script src="/_scripts/site-search.js"></script>

  <script src="/_scripts/tooltip.js"></script>


</head>

  <body>
    







<header class="background" style="--image: url('/images/background.jpg')" data-dark="true">
  <a href="/" class="home">
    
      <span class="logo">
        
          <svg xmlns="http://www.w3.org/2000/svg" viewbox="-40 -60 80 100">
  <style>
    .bubble {
      animation: float 2s ease-out both infinite var(--delay);
    }
    @keyframes float {
      0% {
        opacity: 0;
      }
      50% {
        transform: translateY(0);
        opacity: 0;
      }
      75% {
        opacity: 1;
      }
      100% {
        opacity: 0;
        transform: translateY(-40px);
      }
    }
  </style>
  <g fill="currentColor" opacity="0.5">
    <circle class="bubble" cx="0" cy="-10" r="3" style="--delay: 0.1s"></circle>
    <circle class="bubble" cx="0" cy="-10" r="3" style="--delay: 0.4s"></circle>
    <circle class="bubble" cx="0" cy="-10" r="3" style="--delay: 1.1s"></circle>
  </g>
  <path fill="#38bdf8" d="
      M 0 -22.5
      L -19.5 -11.25
      L -19.5 11.25
      L 0 22.5
      L 19.5 11.25
      L 19.5 -11.25
      z
    "></path>
  <path fill="#bae6fd" d="
      M 0 -22.5
      L -19.5 -11.25
      L 0 0
      L 19.5 -11.25
      z
    "></path>
  <path fill="none" stroke="currentColor" stroke-width="5" d="
      M -18 -53
      L -10 -53
      L -10 -29.2
      L -30.3 -17.5
      L -30.3 17.5
      L 0 35
      L 30.3 17.5
      L 30.3 -17.5
      L 10 -29.2
      L 10 -53
      L 18 -53
    "></path>
</svg>

        
      </span>
    
    
      <span class="title" data-tooltip="Home">
        
          <span>Voight Lab</span>
        
        
      </span>
    
  </a>

  <input class="nav-toggle" type="checkbox" aria-label="show/hide nav">

  <nav>
    
    
      
        <a href="/research/" data-tooltip="Research Interests and Scientific Activities">
          Research
        </a>
      
    
      
        <a href="/team/" data-tooltip="About Our Team">
          Team
        </a>
      
    
      
        <a href="/tools/" data-tooltip="Published Methods">
          Tools
        </a>
      
    
      
        <a href="/data/" data-tooltip="Datasets">
          Data
        </a>
      
    
      
        <a href="/publications/" data-tooltip="Published Works">
          Publications
        </a>
      
    
      
        <a href="/join/" data-tooltip="Join our Team!">
          Join Us!
        </a>
      
    
      
        <a href="/calendar/" data-tooltip="Ben Voight's Public Calendar">
          Calendar
        </a>
      
    
      
        <a href="/contact/" data-tooltip="Email, address, and geolocation">
          Contact
        </a>
      
    
  </nav>
</header>

    <main>
      <!--
  modify main content of page:
  - add section breaks
  - attach section properties
  - wrap each table in div to allow for scrolling
  - filter out blank sections
-->








  
  
  

  <section class="background" data-size="page">
    <h1 id="research-interests-and-scientific-activities">
<i class="icon fa-solid fa-lightbulb"></i>Research Interests and Scientific Activities</h1>

<!-- start popgen unit -->

<div class="feature">
  <a href="/2024/01/11/Projects-Pop-Gen" class="feature-image" aria-label="Population Genetics and Evolution">
    <img src="/images/tools/aggarwala-mutrate.png" loading="lazy" alt="Population Genetics and Evolution" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  <div class="feature-text">
    
      <p class="feature-title">Population Genetics and Evolution</p>
    
    
<p>We are an active population genetics lab and projects focus around studies of mutation rate variation, natural selection, and at the interface between population genetics and complex traits. Projects include both methods development and/or data science analysis. We are especially interested in development and applications using <em>Baymer</em>, a tool to infer mutation rate variation based on local nucleotide sequence context that we recently reported. We are also active collaborators with other Population Genetics minded groups at Penn! (Image by Stacie L. Bumgarner, PhD - Red Windmill Studio)</p>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Pop-Gen" data-style="" data-flip="" aria-label="fa-solid fa-arrow-right">
      <i class="icon fa-solid fa-arrow-right"></i>
      
        <span>Pop-Gen Projects</span>
      
    </a>
  </div>


  </div>
</div>

<!-- end popgen unit -->

<!-- start Complex Traits -->

<div class="feature">
  <a href="/2024/01/11/Projects-Complex-Traits" class="feature-image" aria-label="Dissection of Complex Traits">
    <img src="/images/complex-traits2.png" loading="lazy" alt="Dissection of Complex Traits" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  <div class="feature-text">
    
      <p class="feature-title">Dissection of Complex Traits</p>
    
    
<p>We are interested in the architecture of complex traits, identification of causal genes, and inference of perturbations informed by human genetics predicted to be therapeutically actionable. Our space of primary interest include cardiometabolic traits, including type 2 diabetes, cardiovascular disease, liver disease, and causal traits related to these enpoints. We are also generally interested in pleiotropy at large. We touch all layers, including GWAS, pheWAS, fine-mapping, variant-to-gene mapping, risk prediction via PRS, and collaborations to validate causal genes. (Image credit to Uffelmann et al. 2021, Fig. 3b)</p>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Complex-Traits" data-style="" data-flip="" aria-label="fa-solid fa-arrow-right">
      <i class="icon fa-solid fa-arrow-right"></i>
      
        <span>Complex Trait Genetics Projects</span>
      
    </a>
  </div>


  </div>
</div>

<!-- end Complex Traits unit -->

<!-- start MR unit -->

<div class="feature">
  <a href="/2024/01/11/Projects-MR" class="feature-image" aria-label="Mendelian Randomization">
    <img src="/images/MR.png" loading="lazy" alt="Mendelian Randomization" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  <div class="feature-text">
    
      <p class="feature-title">Mendelian Randomization</p>
    
    
<p>Causal inference is one of the most challenging - but important - problems in biology and medicine. As a human genetics lab, we are active in the application of <em>Mendelian Randomization</em> as a statistical approach to provide one line of evidence on the causal inference problem. Available projects include Big Data analysis or methods development. (Image credit to Sanderson et al. 2022, Fig. 1)</p>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-MR" data-style="" data-flip="" aria-label="fa-solid fa-arrow-right">
      <i class="icon fa-solid fa-arrow-right"></i>
      
        <span>MR Projects</span>
      
    </a>
  </div>


  </div>
</div>

<!-- end MR unit -->

<!-- start Consortium units -->

<div class="feature">
  <a class="feature-image" aria-label="Consortium Level Activities">
    <img src="/images/MVP.webp" loading="lazy" alt="Consortium Level Activities" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  <div class="feature-text">
    
      <p class="feature-title">Consortium Level Activities</p>
    
    
<p>My group is active in several consortia, generally focused around human genomics data. The objectives of these interactions are to either assemble data for discovery, collaborate to advance functional validation of causal genes for T2D, access to large-scale biobank data (e.g., MVP), or to develop new knowledge portals to aggregate data for discovery and inference of disease processes.</p>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Consortia" data-style="" data-flip="" aria-label="fa-solid fa-arrow-right">
      <i class="icon fa-solid fa-arrow-right"></i>
      
        <span>See Our Consortium Partnerships</span>
      
    </a>
  </div>


  </div>
</div>

<!-- end Consortium unit -->

<!-- start Aspirational units -->

<div class="feature">
  <a href="/2024/01/11/Projects-Aspirational" class="feature-image" aria-label="Dreams for Future Work">
    <img src="/images/spiderman.svg" loading="lazy" alt="Dreams for Future Work" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  <div class="feature-text">
    
      <p class="feature-title">Dreams for Future Work</p>
    
    
<p>We also have some exciting and crazy aspirations for other projects, looking for just the right person (and resourcing)!</p>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Aspirational" data-style="" data-flip="" aria-label="fa-solid fa-arrow-right">
      <i class="icon fa-solid fa-arrow-right"></i>
      
        <span>Aspirational Projects</span>
      
    </a>
  </div>


  </div>
</div>

<!-- end Aspirational unit -->

<!-- 





  

  

  
    

    

<div class="post-excerpt">
  
  
  <a href="/2024/01/11/Projects-Pop-Gen.html">Projects in Population Genetics</a>

  <div class="post-info">
  
    
    
      <span data-tooltip="Author">
        <i class="icon fa-solid fa-feather-pointed"></i>
        <span>Ben Voight</span>
      </span>
    
  

  
  

  
    <span data-tooltip="Originally published on">
      <i class="icon fa-regular fa-calendar"></i>
      <span>January 11, 2024</span>
    </span>
  

  
    <span data-tooltip="Last updated on">
      <i class="icon fa-solid fa-clock-rotate-left"></i>
      <span>February 09, 2024</span>
    </span>
  
</div>


  






  
  
  <p data-search="                        Links to Pubs                               Pop Gen                        Mutation Rate                        Selection                        Baymer                        Demography                                    Mutation Rates                                    Selection                                    Interface of Pop Gen Complex Traits            Inference of Mutation Rate Variation across GenomesMutation is one of the most important processes generating variation within and between species  and is a fundamental process underlying molecular evolution  My lab is generally interested in work to identify correlates of   and ways to ultimately explain   variation in where  what type  and how frequent different types of mutational events occur in genomes  polymorphisms  substitutions  de novo events  somatic mutations  etc  Ultimately  better models to explain variation in mutation rates across genomes can help interpret evidence for selection  e g   functional constraint   timing and the pace of the molecular evolutionary clock  how the process of mutation has changes over time or may be different across populations or across the tree of life broadly  Moreover  better models may also ultimately help us to understand mechanism  i e   what are the ultimately environmental and molecular drivers underlying the mutational process To date  our work has mainly focused on modeling variation in mutation rates using sequence context   the local nucleotides that flank a mutated site  For example  it is well known that the rate of C gt T transition mutations are substantially elevated at CpG dinucleotide sites owing to the mechanism of spontaneous deamination of 5 methylcytosine  This observation serves as the foundation for questions and models that we explore  namely   How much variation in where  what type  and how frequent different types of mutations can be captured by sequence context   Can improved models for mutation rate variability help identify regions of functional constraint   How has the mutational process   measured by expanded windows of sequence context   changed over time Below  I describe a couple of ideas of interest  but generally I am open to different types of analysis projects or ideas in this space   Further Development of Baymer  Our primary report for Baymer describes a  baseline  implementation that is poised for further development  Some possible angles for novel development include  but not limited to            Cross substitution class modelling  As currently implement  Baymer stratifies the data into 1 mer models  A gt G  A gt T  A gt C  etc    It is possible that information about rates and local sequence context could be borrowed across all types       Multi population models  When we apply Baymer  we typically do so with by grouping individuals based on some notion of ancestry or similarlity to reference groups  but each separately  However  we know that the history of human populations is connected and can be described by phylogenetic trees with mutations shared across populations  private to specific branches  or may migrate across populations  Extensions to Baymer that consider such models could be imagined which fully utilize all data collectively  rather than stratifying  and could be used to make comparisons of rates across populations       Formal estimation of θ  Baymer quantifies the probability of mutation for a given window of sequence context  based on the total number of mutated sites observed divided by the total number of k mer sequence contexts in the reference  or reconstructed ancestral  genome  However  this basic model has limitations and assumptions which we acknowledge  First is that this probability depends on  but is not itself an estimate of the base pair per generation mutation rate   mu   It is also true that this probability depends on several additional quantities including  i  effective population size and other aspects of demographic history that influence patterns of genomic variation   ii  forces that may influence patterns of variation locally  e g   selection and recombination   iii  biological or environmental factors that influence the mutation rate  either globally or locally  Thus  a direction for work would be to improve Baymer with explicit models of these forces to generate estimates of θ       Models for recurrent mutation events  mutation saturation  Baymer makes an infinite sites model  but as the number of surveyed genomes increase  mutational contexts can become saturated  We also know that highly mutable sites that may not yet be saturated can be subjected to recurrent mutational events as well  Work to relax this assumption and accomodate these factors into a modified modeling procedure could be imagined             Expanded model development for non sequence context features  Other  non sequence contexts  correlates of mutation rate   Replication timing  rate of recombination  strand biases  functional genomics  etc  These factors absolutely contribute to variation in where  what type  and how frequent polymorphisms are observed and may  or may not  correlate with sequence context features  Another goal would be to select key factors and incorporate them into a unified Baymer model which describes variation across sequences context and one or more of these features         Applications of Baymer   Across the Tree of Life  A central question is to what extent are mutational processes conserved across the tree of life  The reduction of cost for large scale sequencing has made creation of draft genomes of thousands of species now possible and facilitates the collection of population level variation data in those species  As a result  it is extremely timely to bring extant data   including but beyond mammals   to this basic question  Our lab is collaborating with the Mathieson Lab to cultivate  gt 100 species worth of polymorphism data to characterize mutational profiles across species  to determine if patterns of variation are different  differently evolving  and if we can correlate features common to species with mutational profiles or spectra  given the structure of the phylogenetic tree     Applications of Baymer   Coding Genome  Previous work focused on the non coding genome  we expect that patterns of variation in the coding genome  incorporating codon context with sequence context window  should reflect what is observed in the non coding genome  albeit reduced as a function of purifying selection  The approach would be to improve Baymer to be knowledgable about codon context and strand bias  directionality  to evaluate this model  In particular  the field uses synonymous mutation to generate an expectation for neutral  background  variation  Yet  a small fraction of these sites might still be subject to constraint  e g   cryptic splice enhancer modifiers   Thus  the ability to generate an appropriate background based on the genome background would facilitate better models to test all types of variation  Moreover  we also have substantially larger and better refined information on protein structure than ever before   generated empirically from Cryo EM or through computational predictions e g   AlphaFold  AlphaFold2  Another objective is to better isolate regions in proteins likely to be the most constrained overall  to discover regions that are functionally important                         Back to Top            Inference of SelectionI have a longstanding interest in discovery of genomic loci that may be targetted by directional positive selection or adaptive evolution in human populations  Ultimately  our goals are to built new and improve upon existing method to identify genomic signatures in data consistent with the action of different types of selection  I am also fond of much harder work to identify and characterize genes subject to selection that may arise from these efforts  connecting them to human traits to try to understand what may have been important in evolutionary history  primarily in human populations  Previous students in the lab developed new summary statistics to detect the action of balancing selection at genomic loci  the β Statistics   or improved upon and applied existing tools  the iHS  to multi population genetics data  More recently  I am interested in applying our work to model mutation rate variation as a function of sequence context to improve resolution of sequences   coding and non coding   subject to purifying selection  and the strength of that selection on those sequences Some current interest include       Applying Baymer and context models for new measures of functional constraint  extended regions of homozygosity  adaptation  The outputs from Baymer include the probability to observe polymorphism at a given nucleotide sites from population data  Using this information  it is possible to generate statistics that quantify the difference between observed vs  expected levels of polymorphism in specific genomic regions relative to the genome background as a whole  For example  we expect to see a reduction of polymorphic sites the coding relative to the non coding genome  owing to functional constraint on coding sequences  Or  for example  directional selective sweeps that have have only recently fixed in a population will not have had much time to generate new mutations in the region  thus  one expects under some models to observe a deficit absence of segregating sites in a  completed  sweep  e g   SLC24A5 in Europeans   Thus  projects include developing new test statistics based on this output to look for patterns in genomes consistent with different modes of selection         Baymer   reinterpret previously pop gen statistics with better models for mutation rates  Many of the population genetic summary statistics that I grew up learning about that were relevant for detection of selection   the MK test  HKA  dn ds  etc  are based fundamentally on models for the mutational process  Generally  those models make limited assumptions about sequence context  these could be re evaluated generally in the light of empirical data and evidence supporting variation in context based mutability  within and across species          Application of selection scans to multi species data  In the process of assembling multi species data for mutation rate inference using Baymer  depending on the quality of the genome assembly and the data collected  it may be possible to apply some of our methods to species beyond humans  to identify signatures of selection from these data                           Back to Top            Projects at the Interface between Population Genomics and Complex Trait GeneticsMy previous training spans population genetic and complex disease trait mapping  I view these as complementary  population genetic thinking is quite informative  infectious  even  to interpreting complex trait association data  deciphering the genetic architecture of complex traits  Along these lines  we try to borrow ideas from population genetic models to understand disease relevant variation or prediction problems  or how to develop and improve upon statistical genetics approaches that take advantage of multi ancestry studies across human populations  My lab is always on the hunt for interesting angles where ideas from both spaces can be blended into interesting project  Some examples are listed below       Somatic mutational data in cancer  The process and accumulation of somatic mutations is integral to the development of cancer in various tissues  and a great deal of prior work has been performed to learn about mutational signatures that are observed from these data as well as attempting to connect those signatures to molecular processes  However  most of these models are in 3 mer context models  Large scale somatic mutational data are available to which Baymer could easily be applied to discover new signatures from data  or to use outputs from Baymer  and other measure of selection constraint  to clarify somatic mutations that may be pathogenic         Functional constraint models to interpret coding  non coding variation for complex traits  While a great deal of functional constraint is observed and relatively well quantified in the coding genome  quantification of constraint in the non coding genome is much less obvious  There is a great deal of new methods and new data work from multi species  mammalian  data  e g   primate sequencing  Zoonomia  that is bringing unprecedent data and resolution to infer functional constraint  We think potentially there are gains to be made applying functional constraint inferred from measurements informed by sequence context mutational probabilities inferred by Baymer to quantify constraint further  This could then be applied to interpret human complex trait association data  particularly as output from statistical signal fine mapping  where additional information about variants within a credible set are likely to be causal would be exceedingly valuable to help guide functional follow up experimental efforts         Population Structure  admixture and quantitative genetics  We are engaged in large scale DNA biobanks like the Million Veteran Program  of whom the participants are population genetically diverse  This provides the opportunity to better understand the population genetics of individuals from those biobanks  to infer and quantify admixture to improve genetic mapping effort  and to understand complex disease architecture better  In addition  inferring community level information about sub population structure can be very important to identifying pathogenic variation that is at elevated frequency in those groups                           Back to Top            ">
    

    
      
      
        Links to Pubs:
      
    
  


    
      
        Pop Gen
      
    
      
        Mutation Rate
      
    
      
        Selection
      
    
      
        Baymer
      
    
      
        Demography
      
    
  


  </p>
</div>

  
    

    

<div class="post-excerpt">
  
  
  <a href="/2024/01/11/Projects-MR.html">Projects in Mendelian Randomization</a>

  <div class="post-info">
  
    
    
      <span data-tooltip="Author">
        <i class="icon fa-solid fa-feather-pointed"></i>
        <span>Ben Voight</span>
      </span>
    
  

  
  

  
    <span data-tooltip="Originally published on">
      <i class="icon fa-regular fa-calendar"></i>
      <span>January 11, 2024</span>
    </span>
  

  
    <span data-tooltip="Last updated on">
      <i class="icon fa-solid fa-clock-rotate-left"></i>
      <span>February 09, 2024</span>
    </span>
  
</div>


  






  
  
  <p data-search="                        Links to Pubs                               Mendelian Randomization                                    MR Analysis                                    MR Methods            OverviewCausal inference is one of the most challenging problems in biology and medicine  requiring multiple lines of evidence supported by model systems  pharmacology  and genetics to demonstrate convincingly  Under specific assumptions  it is possible to use human genetic variation as an instrumental variable to estimate the causal effect of an exposure on an outcome  For example  loss of function variation in PCSK9 lowers LDL cholesterol  LDL C  and susceptibility to heart disease  consistent with the fact that elevated LDL C is a causal risk factor for heart attack  There is much nuance here  with advances in new methods  understanding the testable  and untestable  assumptions of the model  kits of tools that make application very straight forward   a complete didact on this subject is well beyond the purpose of this short overview  For those who are interested  I strongly recommend picking up a recently published MR Primer                                       Mendelian randomization              Sanderson E   Glymour MM   Holmes MV   Kang H   Morrison J        Palmer T   Schooling CM   Wallace C   Zhao Q   Smith GD              Nature Reviews Methods Primers       nbsp     nbsp       10 Feb 2022       nbsp     nbsp       doi org 10 1038 s43586 021 00092 5                                Mendelian Randomization AnalysesWe are generally interested in novel relationship discovery as well as clarifying previously observed exposure outcome relationships  particularly  but not exclusive to  the cardiometabolic trait space  For example  we have explored the role of smoking and body mass index  BMI  on T2D risk  serum urate on CAD T2D  and BMI and Waist Hip ratio  WHR  on blood traits  Ongoing   active projects include       Nutrition   dietary exposure on cardiometabolic traits disease  Dietary preference is conventionally thought of as an environmental risk factor  but previous studies have elucidated a significant  but small  heritable component to this collection of traits  This facilitates MR experiments to understand what  if any  role this  environmental  exposure has on T2D  CVD  or related causal traits  A leading hypothesis is that risk from dietary preference to outcome is mediated through anthropometric factors like adiposity  This work is led by Mia Lee  PGG graduate student  in the lab         Phenome wide MR  considerations for diverse ancestry analysis   In collaboration with Drs  Venexia Walker and Tom Gaunt  Univ  of Bristol  UK  and Dr  Scott Damrauer  Penn   we are engaged in a broad stroke  all by all MR experiment  i e   hundreds of exposure outcome pairings  via summary data from MVP and the UKBB  This will provide a deep  well powered resource for discovery  but will explore the role and extent of biases that must be contended with in the case of ancestry mismatching         Collaborative activities with T2D GGI  Collaborating within the T2D GGI  we are engaged in the latest GWAS for T2D and QTL data through which we can  drug target  via QTLs  The goal with this approach is to identify potential leads and directions of effect for changes in molecular phenotype  e g   expression  with a target gene  to make an educated guess at the predicted effect that would be therapeutic  In addition  work is underway which seeks to relate our previously defined genetic clusters to T2D comorbidities  e g   retinopathy  stroke  etc   through the MR framework   Generally  we are open to different types of analysis projects that are along the lines of specific targeted questions                         Back to Top            Mendelian Randomization MethodologyWe are also interested in developing methods in MR studies  Previously  we have developed pipelines and a simulation engine to evaluate  empirically  MR under different models and assumptions Methods development for MR is a particular challenge  broadly because the field moves rapidly and there are a great many biostatistician epidemiologists who have dedicated their entire operation to MR  and methods development   Where possible  we seek to bring in these latest methods into ongoing work  Occasionally  we identify methodologic  targets of opportunity  while working with Big Data sets or by pushing the envelope of human genomics studies  We have a couple of areas of interest in this space  primarily along the lines of instrument curation  and when considering large scale data sets which derive from a plurality of ancestries  That said  my group is certainly open to other lines of inquiry  so interested parties should express their interest so that we can discuss further                         Back to Top            ">
    

    
      
      
        Links to Pubs:
      
    
  


    
      
        Mendelian Randomization
      
    
  


  </p>
</div>

  
    

    

<div class="post-excerpt">
  
  
  <a href="/2024/01/11/Projects-Complex-Traits.html">Projects in Complex Trait Genetics</a>

  <div class="post-info">
  
    
    
      <span data-tooltip="Author">
        <i class="icon fa-solid fa-feather-pointed"></i>
        <span>Ben Voight</span>
      </span>
    
  

  
  

  
    <span data-tooltip="Originally published on">
      <i class="icon fa-regular fa-calendar"></i>
      <span>January 11, 2024</span>
    </span>
  

  
    <span data-tooltip="Last updated on">
      <i class="icon fa-solid fa-clock-rotate-left"></i>
      <span>February 09, 2024</span>
    </span>
  
</div>


  






  
  
  <p data-search="                        Links to Pubs                               Complex Traits                        T2D                        MASLD                        CAD CVD                        QTL Mapping                                    T2D                                    MASLD                                    QTL Mapping                                    Additional Collaborations            Discovery and Analysis  Type 2 Diabetes and its ComplicationsI have a longstanding interest in dissection of the genetic basis of type 2 diabetes  T2D   A great deal of progress has been made in characterizing the basis of the disease  in no small part to the advent of genome wide association studies  GWAS  and the scale to which data can be collected  My efforts over my career thusfar has contributed to data collection and analysis that has resulted in the discovery of hundreds of associated loci to date  While work continues to expand discovery of new loci for the disease  expanding especially towards characterization of genetic liability across diverse cohorts  a great deal of work still remains  post  genome wide association study  My fundamental drive is to identify genes and pathways that are therapeutically actionable  to expand the collection of treatments available  perhaps personally tailored to the sub structure of genetic risk to T2D that an individual might carry  I believe genomics data collected in humans is central to that effort  identifying relevant leads  the therapeutic direction of effect  the  horsepower  of specific human genetic targets  and so forth  As such  there are a range of GWAS and post GWAS activities that are of active interest in the lab  with project invariably collaborative and large scale with the T2D GGI and MVP consortia  To enumerate a couple of areas interest       Locus Discovery  Our most recent work has reported the assembly  gt 2 5M participants for the largest multi ancestry GWAS of T2D to date  which lead to 145 newly loci for disease and bring the number of distinct loci to 611 and signals to 1289  However  this still did not bring the complete collection of data to bear  Ongoing data collection continues with AllofUs  MVP  and potentially bringing in cohorts to cover population groups that were not as well represented in this new effort as would have been desired  e g   from South Asia          Signal Fine Mapping  Mapping causal variants is directly relevant to causal gene identification  understanding mechanisms  and disease progression  Signal fine mapping is the appropriate statistical method to identify multiple  statistically independent signals associated  and is important as it   i  extends the heritability explained by genetics  allowing us to deduce the relevance of causal variants shared across   but also potentially unique to   ancestry groups   ii  allows us to connect association signals to additional metabolic and progression traits directly  and  iii  provides additional leads to connect to a putative causal nearby gene In our initial efforts for T2D GGI  we applied fairly coarse  heuristic  but not quantitative  approach to capture putatively independent signals of association at each distinct T2D associated locus  However  the toolkit of available approaches and data sets to deeply characterize the full spectrum of allelic heterogeneity comprehensively have never been better  I am particularly interested in application of these tools for T2D GGI  to bring together true  in sample  LD reference panels for large biobanks with summary data they contributed to meta analysis to attack this problem         Causal inference  Within the T2D GGI  collaborations are ongoing to use the framework of Mendelian randomization  MR   QTL data  and multi ancestry information to triangulate on variants and their predicted effects on nearby genes  to identify direction of effects and  horsepower  of given perturbations of that gene to risk to T2D that might be expected  In addition  I am also interested in understanding non linear effects of established causal risk factors and lifestyle factors in the context of T2D genetic liability  These questions can be addressed with the framework of MR  but require individual level data   which is available in MVP  Use of these data potentially further facilitates exploration of the role of causal risk factors for T2D across diversity ancestries  and to relate those risk factors potentially to risk of complications of T2D         Colocalizaton  Variant to Gene Mapping and multi trait physiology  It is highly desirable to connect non coding variants   the lion s share of tagged association at T2D risk loci   with a cognate  putatively relevant gene in cis  This can be achieved computationally through varaint to gene mapping approaches  e g  using variation associated with change in transcript abundance  eQTLs  in relevant tissues or cell types  e g   beta cells of pancreatic islets   We are interested in attacking the variant to gene mapping problem generally  through analysis of existing data as well as generating new QTL data sets for this purpose   but also in the context of credible sets identified from signal fine mapping efforts  In addition  we are also interested in attaching the complete spectrum of physiologic associations at each signal of association  We are interested in this approach  to  i  help understand the traits that might be expected to be perturbed  which may be beneficial or antagonistic to therapeutic indications    ii  as a vehicle to help characterize pleiotropy generally in the genome  and  iii  to potentially obtain leads for the potential tissue or cell type of action that may be physiologically implicated at the associated signal         Genetic Sub Typing via Trait Clustering  In our recent work  and consistent with reports from others  it is possible to group T2D signals by cardiometabolic and anthropometric trait associations at those loci  These clusters appear to map onto to distinctive functional genomic and tissue groupings  e g   beta cell biology vs  adipose   obesity biology  In our paper  we demonstrated initial evidence that these sub types may also diffferent associate with risk of T2D complications  though clearly more work to fuel stratification are necessary  In additional collaborative work within the T2D GGI  we are exploring and contrasting different approaches to clustering  and are interested in evaluating new approaches to clustering fueled by additional data or new ways to approach the problem  e g   colocalization or fine mapping   The objective is potentially to develop and identify types that predict downstream complications  e g   CAD  Kidney disease  retinopathy  etc    which could potentially be transformative if successful                           Back to Top            Discovery and Analysis  Metabolic associated steatotic liver disease and its ProgressionThrough involvement with MVP  I have recently become interested in dissecting the genetic basis of metabolic associated steatotic liver disease  MASLD  previously referred to as NAFLD  and its progression  Our efforts involve not just locus discovery for the trait  or proxy  and progression phenotypes  but also post GWAS analyses  and is a collaborative activity with many investigators at Penn and the VA  including Kyong Mi Chang and Marijana Vujkovic  among many others       Locus Discovery  Following along our previously work  efforts continue to map new susceptibility loci for the disease  or for related traits like liver cancer or cirrhosis  As data sets continue to grow  we are intrested in aggregating them for novel discovery as well as for down stream analyses         Signal fine mapping  Mapping causal variants is directly relevant to causal gene identification  understanding mechanisms  and disease progression  Signal fine mapping is the appropriate statistical method to identify multiple  statistically independent signals associated with disease  These allow us to extend heritability explained as well as find new targets for variant to gene mapping  As with T2D  I am particularly interested in application of novel tools and applications in MVP to deeply characterize the full spectrum of allelic heterogeneity comprehensively have never been better  The collection of liver disease and progression traits are opportune target for these efforts         Multi Trait mapping  MASLD by new definition carries with it at least one cardiometabolic trait association  I am interested in characterizing the trait physiology perturbed at MASLD and progression traits  identifying loci that are associated with lean MASLD  and performing multi trait GWAS to expand the catalog of relevant associations that could be targets for functional studies beyond our primary scan         Functional validation of MASLD Loci in the Zebrafish model system  Of established MASLD associations  a few transcripts identified through GWAS have been assigned causality for relevant phenotypic traits  such as liver fat  injury  or fibrosis cirrhosis  These transcripts harbor variants with positive or negative risk associations that act through loss  and gain of function mechanisms in either hepatocytes or communicating cell types  immune   But like other complex diseases  many of the leading variants associated with MASLD are non coding  which do not demarcate a clear candidate gene  While computational approaches can narrow down the list of potential candidates to reduce the number of genes in the list  candidate genes nominated in this way still require functional validation to demonstrate a phenotypic effect  direction of effect  and underlying mechanisms  Because we have dozens of MASLD candidate genes to evaluate  we need in vivo model system that facilitates tractable throughput  is scalable  and yet retains phenotypic depth and relevance  As such  we are embarked to build a collaboration with den Hoed Lab  Uppsala  to use zebrafish as a model system for validation  Zebrafish have a well annotated genome with orthologues of at least 71  of human genes  or 82  of human disease associated genes  Our results so far show strong directional consistency in effects on liver fat across zebrafish larvae and humans or mice for genes previously implicated through pLOF variants                           Back to Top            Quantitative Trait Locus Mapping ActivitiesWe have recently expanded our interest and expertise in performing genetic mapping of molecular phenotypes  i e   quantitative trait locus  QTL  discovery  Previously  we have performed scans in cell models for transcript abundance  eQTLs  or proportion of alternative spliced isoforms  sQTLs   and collaborate with local investigators to study primary tissues  including kidney  with Katalin Susztak   More recently  we are involved in several projects which utilize primary tissue in bulk and at single cell resolution  as well as expanding to chromatin accessibility  caQTL  mapping  Active projects include       Single cell eQTL and caQTL mapping for islet cell types  In collaboration with Gaulton Lab  UCSD  and Grant Lab  Penn CHOP   led by shared graduate student  Mary Ann Weidekamp   we are using single cell data generated by the Human Pancreas Analysis Program which is collecting multi omic readouts of pancreatic tissue obtained from deceased donors  These data include scRNA Seq but also scATAC Seq  and more recently  multiome RNA ATAC seq readouts  Our initial focus has been on mapping eQTLs in alpha  beta  and delta cell populations  as well as additional cell types also captured in this experiment  e g  acinar cells          QTL mapping of B T cells in pediatric cohort  In collaboration with Romberg Lab  CHOP   we are engaged in eQTL mapping in matched immune cell types  T cell  B cell  obtained from tonsil tissues extracted from pediatric cohorts  under baseline and stimulated conditions  These data are particularly novel  as they characterize immune cell populations from within a tissue context  not circulating in Blood  i e  PBMCs   and derived from a pediatric cohort  age  lt 18   Our initial efforts in  gt 100 subjects have mapped  14 000 eGenes in basal and stimulated states  including  gt 1 700 not previously reported in either large scale data sets  i e   GTEx or eQTLGen  nor from catalogs of eQTLs from immune cell type data obtainde from PBMCs  i e   the DICE consortium          caQTL mapping in Bulk Liver  Lead by a graduate student  Bradon Wenz  during his time working with Casey Brown  we are focused on the analysis of  180 primary bulk liver samples that he profiled using chromatin accessibility mapping via ATAC Seq  The result is a massive map of regions of accessible chromatin in bulk liver tissue as well as a catalog of caQTLs  We are working together to finalize the analyses for these data                           Back to Top            Additional Collaborative ProjectsWe also have a number of ongoing projects that generally involve collaborations or shared graduate students with another lab with complementary expertise our lab       Fine mapping and functional validation in Bone mineral Density  Genetic factors strongly influence bone mineral density  BMD  and osteoporosis risk over time  To date  genome wide association studies  GWAS  have discovered over 1 100 independent BMD signals  however  the causal variant s  and respective effector gene s  for the majority of these loci remain unknown  In collaboration with the Grant Lab  we previously published the results of a 3D genomics based  variant to gene  mapping method for BMD loci based on ATAC seq  RNA seq  and chromatin confirmation capture data from hMSC derived osteoblasts  Led by graduate student  Mitch Conery  co advised by Struan and myself  we are applying this approach  which links putative causal variants in open chromatin to candidate effector genes via chromatin interactions with open promoters  to a dataset derived from the human fetal osteoblast 1 19 cell line  hFOB   For validation of both sets of findings at scale  we conducted a CRISPRi screen in hFOBs differentiated for five days  In addition  Mitch is performing multi trait signal fine mapping for these signals across a range of related traits  to inform further prioritization efforts and potentially link BMD signals to novel biological and physiological mechanisms         Genome wide  phenome wide Association studies in MVP  I was fortunate to join colleagues nation wide  to report a genome wide  Phenome wide association study across MVP participants  The scope and volume of computation require to complete this project require collaborations with the DOE and supercomputer access with modified  efficient implementations of analysis tools  leading to new collaborations and inference at scale that has hitherto not been reported  There  across  gt 2 000 traits in  gt 630K MVP participants  we reported  gt 38 000 independent variants associated with 1 or more traits  performed fine mapping which resulted in 6 318 singals from 613 traits to single variant resolution  Future work extends in this vein to involve X Chromosome analysis and expanded fine mapping using multi ancestry aware tools like SuSiEx  meSuSiE  mJAM  or flashfm                           Back to Top            ">
    

    
      
      
        Links to Pubs:
      
    
  


    
      
        Complex Traits
      
    
      
        T2D
      
    
      
        MASLD
      
    
      
        CAD/CVD
      
    
      
        QTL Mapping
      
    
  


  </p>
</div>

  
    

    

<div class="post-excerpt">
  
  
  <a href="/2024/01/11/Projects-Aspirational.html">Aspirational Projects</a>

  <div class="post-info">
  
    
    
      <span data-tooltip="Author">
        <i class="icon fa-solid fa-feather-pointed"></i>
        <span>Ben Voight</span>
      </span>
    
  

  
  

  
    <span data-tooltip="Originally published on">
      <i class="icon fa-regular fa-calendar"></i>
      <span>January 11, 2024</span>
    </span>
  

  
    <span data-tooltip="Last updated on">
      <i class="icon fa-solid fa-clock-rotate-left"></i>
      <span>February 09, 2024</span>
    </span>
  
</div>


  






  
  
  <p data-search="                        Links to Pubs                               Spiders                        Docking                                    Spiders                                    Docking            Spider GenomicsSpiders are an amazing species  most notably for producing biomaterials that have some of the most unique and diverse biophysical properties of materials found in nature  Our previous work focused on the genomic assembly and silk gene characterization of two species of spiders  the Golden Orb weaver Trichonephila clavipes  one of the most well studies arachnids for silk biology  and Caerostris darwini  a species native to Madagascar infamous for its extremely tough draglines used in the webs that it builds over riverways there for predation We remain interested in characterizing these  and other  related spider species  as sequencing technologies develop and advance  For example  some project directions could include creating a more  complete  Telomere to Telomere genome  collecting population genetic data to characterize demographic history  variation in mutation rates  and detecting signature of natural selection  or using advances sequencing technologies to characterize transcript expression in silk glands at single cell and or at spatial resolution                               Golden Orb Weaver Spiderlings in the lab                                           Back to Top            Computational Prediction of Molecular DockingA great deal of effort in human genetics is focused on solving the  variant to gene  mapping problem  namely  identifying what gene at a given locus is causal for the phenotype disease that is associated with regional variation  However  once a leading gene has been identified and proven to be implicated in disease beyond a reasonable doubt  typically with functional data from cell or animal model systems data   it is natural to ask if the protein product of that gene can be perturbed with a small molecule to obtain a therapeutic effect  While this direction of experimental effort is often considered an affair for the private sector  development of  tool  small molecules  precursors for true therapeutics that demonstrate potential effect  is still a risky proposition  Moreover  biochemical perturbation of gene targets that are consistent with the observed human genetics phenotypic data provides another line of evidence for gene relevance  even if the target is not fully actionable While a substantial computational task  if sufficiently high resolution protein structures for a target exists  there are tools that can model the proclivity of a small molecule to  dock  with the surface of that structure  We are interested in identifying targets of opportunity  namely  for cardiometabolic traits of lab interest where we have implicated a gene beyond reasonable doubt  to determine which  if any  of those targets are amenable to computational docking experiments  Screening millions of compounds takes computational time  so we are also interested in developing computationally efficient pipelines to take this task on at scale                         Back to Top            ">
    

    
      
      
        Links to Pubs:
      
    
  


    
      
        Spiders
      
    
      
        Docking
      
    
  


  </p>
</div>

  
    

    

<div class="post-excerpt">
  
  
  <a href="/2024/01/11/Consortia.html">Consortium-Level Activties</a>

  <div class="post-info">
  
    
    
      <span data-tooltip="Author">
        <i class="icon fa-solid fa-feather-pointed"></i>
        <span>Ben Voight</span>
      </span>
    
  

  
  

  
    <span data-tooltip="Originally published on">
      <i class="icon fa-regular fa-calendar"></i>
      <span>January 11, 2024</span>
    </span>
  

  
    <span data-tooltip="Last updated on">
      <i class="icon fa-solid fa-clock-rotate-left"></i>
      <span>February 09, 2024</span>
    </span>
  
</div>


  






  
  
  <p data-search="                        Links to Pubs                               PanKbase                        T2D GGI                        T2D AMP CMD                        MVP                                    T2D GGI                                    MVP                                    T2D AMP CMD                                    PanKbase            The Type 2 Diabetes Global Genomics Initiative  T2D GGI I have a long standing commitment to mapping genetic susceptibility loci for type 2 diabetes  T2D   I was part of the first groups to perform GWAS successfully   our efforts there to bring together data initially to make statistically robust  and replicated  observations formed the basis of genome wide meta analyses that we currently enjoy conventionally today  For T2D  I was part of DIAGRAM  the first consortium for meta analysis of T2D data  Today  with long standing colleagues  Andrew Morris and Ele Zeggini   I co lead the Type 2 Diabetes Global Genomics Initiative  T2D GGI   which carries the torch of that previous effort  Our initial effort has been to assemble  gt 2 5M participants for the largest multi ancestry GWAS of T2D to date  Ongoing efforts of that group include use of those data to connect to gene and physiology  through statistical colocalization   refinement and characterization of genetic subtypes of diabetes through clustering methods  causal inference studies via Mendelian randomization  or signal fine mapping  We are a collaborative  inclusive group and projects that utilize data in this area can give trainees exposure to and experience with fantastically effective large scale multi institutional consortium work  which is increasingly typical in human genomics activities                         Back to Top            The VA Million Veteran ProgramThe VA s Million Veteran Program  MVP  is a national research program looking at how genes  lifestyle  military experiences  and exposures affect health and wellness in Veterans  Since launching in 2011  1 million Veterans have joined MVP  It is the largest research effort at VA to improve health care for Veterans and one of the largest research programs in the world studying genes and health  I have a without compensation  WOC  appointment at the local VA here in Philadelphia  CMC   connected to Cardiometabolic Trait and Disease projects funded projects  to Kyong Mi Chang at Penn  and Phil Tsao  Stanford  as well as projects related to CVD PAD  to Scott Damrauer   My primary contribution focused around genetic discovery and analysis for T2D  liver disease  and CVD  This resource is particularly exciting to me with numerous opportunities for projects and productive scholarship  as its highly diverse representation at large sample size in a single repository make it ideal for complex trait genetic and population genetic discovery  characterization of genetic architecture  but also for causal inference using Mendelian randomization approaches that also have the opportunity to work at the individual level  To give one example of the potential of this  I was fortunate to join colleagues nation wide  to report a genome wide  Phenome wide association study across MVP participants  The scope and volume of computation require to complete this project require collaborations with the DOE and supercomputer access with modified  efficient implementations of analysis tools  leading to new collaborations and inference at scale that has hitherto not been reported                         Back to Top            The Type 2 Diabetes   Accelerating Medicines Partnership   Common Metabolic Disease   Functional Genomics Project  T2D AMP CMD FGP Large scale genome wide association studies  GWAS  have lead to the rapid identification of hundreds of T2D associated loci  However  the mechanism s  through which most of these loci influence disease susceptibility remain poorly understood  An interdisciplinary team at Penn  including Pat Seale  Wenli Yang  Struan Grant  Klaus Kaestner  Dan Rader  and myself  brings together experts in population genetics  T2D GWAS  biostatistics  metabolic tissue biology  human cellular disease modeling and T2D pathophysiology to tackle this critical knowledge gap  In collaboration with our consortium partners  our goal is to provide the diabetes research community with a robust pipeline for mapping T2D GWAS variants to effector genes and target tissues  identify new genes and biological pathways that modulate susceptibility to T2D  and define gene regulatory networks relevant to T2D with the goal of uncovering therapeutic  entrypoints  for developing new treatments  My efforts in this consortium lead on genetic discovery and characterization  closely partnering with groups focused on prioritizing putative effector transcripts using QTL mapping data  We also collaborate locally with Penn investigators to help interpret genetics data and to priortize leads that merit functional follow up studies                         Back to Top            PanKbase  Human Islet Research Network  HIRN  Pancreas Knowledgebase Program  PanKbase A wealth of data has been generated in human pancreas donors in initiatives supported by the Human Islet Research Network  HIRN  which can be used to address open questions in type 1 diabetes  yet these data are currently both underutilized and in formats inaccessible to many researchers which prohibits insights  To address this gap  the NIDDK has assembled two groups of highly accomplished researchers to create a pancreatic knowledge base   PanKbase   which leverages expertise in computational biology and data science  type 1 diabetes  islet biology  knowledge base engineering  and engagement and outreach  This is an incredibly new collaboration  as of Feb 2024   I expect more details to emerge about the goals  objectives  and deliverables for the project as the group organized and gets situated                         Back to Top            ">
    

    
      
      
        Links to Pubs:
      
    
  


    
      
        PanKbase
      
    
      
        T2D-GGI
      
    
      
        T2D-AMP-CMD
      
    
      
        MVP
      
    
  


  </p>
</div>

  

 -->
  </section>


    </main>
    


<footer class="background" style="--image: url('/images/background.jpg')" data-dark="true" data-size="wide">
  <!--
    <div>
      Extra details like contact info or address
    </div>
  -->

  <div>
    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="mailto:bvoight28@gmail.com" data-tooltip="Email" data-style="bare" aria-label="Email">
      <i class="icon fa-solid fa-envelope"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://orcid.org/0000-0002-6205-9994" data-tooltip="ORCID" data-style="bare" aria-label="ORCID">
      <i class="icon fa-brands fa-orcid"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://scholar.google.com/citations?user=FexB61cAAAAJ" data-tooltip="Google Scholar" data-style="bare" aria-label="Google Scholar">
      <i class="icon fa-brands fa-google"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://github.com/bvoightlab" data-tooltip="GitHub" data-style="bare" aria-label="GitHub">
      <i class="icon fa-brands fa-github"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://twitter.com/bvoight28" data-tooltip="Twitter" data-style="bare" aria-label="Twitter">
      <i class="icon fa-brands fa-twitter"></i>
      
    </a>
  </div>


    
  </div>

  <div>
    © 2024
    Voight Lab
      |   Built with
    <a href="https://github.com/greenelab/lab-website-template">
      Lab Website Template
    </a>
  </div>

  <input type="checkbox" class="dark-toggle" data-tooltip="Dark mode" aria-label="toggle dark mode" oninput="onDarkToggleChange(event)">
</footer>

  </body>
</html>
