<!DOCTYPE html>
<html lang="en" data-dark="false">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <!--
  put your analytics (e.g. Google Analytics) tracking code here
-->

  <!--
  put your search engine verification (e.g. Google Search Console) tag here
-->

  


























<meta name="viewport" content="width=device-width, initial-scale=1">

<title>Projects in Complex Trait Genetics | Voight Lab</title>

<link rel="icon" href="/images/icon.png">

<meta name="title" content="Projects in Complex Trait Genetics">
<meta name="description" content="An engaging 1-3 sentence description of your lab.">

<meta property="og:title" content="Projects in Complex Trait Genetics">
<meta property="og:site_title" content="Voight Lab">
<meta property="og:description" content="An engaging 1-3 sentence description of your lab.">
<meta property="og:url" content="">
<meta property="og:image" content="/images/share.jpg">
<meta property="og:locale" content="en_US">

<meta property="twitter:title" content="Projects in Complex Trait Genetics">
<meta property="twitter:description" content="An engaging 1-3 sentence description of your lab.">
<meta property="twitter:url" content="">
<meta property="twitter:card" content="summary_large_image">
<meta property="twitter:image" content="/images/share.jpg">


  <meta name="author" content="Ben Voight">
  <meta property="og:type" content="article">
  <meta property="og:updated_time" content="2024-10-14T21:55:08+00:00">
  <meta property="article:published_time" content="2024-01-11T00:00:00+00:00">
  <meta property="article:modified_time" content="2024-10-14T21:55:08+00:00">
  <meta name="revised" content="2024-10-14T21:55:08+00:00">


<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "BlogPosting",
      "author": { "@type": "Person", "name": "Projects in Complex Trait Genetics" },
      "datePublished": "2024-01-11T00:00:00+00:00",
      "dateModified": "2024-10-14T21:55:08+00:00",
    
    "name": "Projects in Complex Trait Genetics",
    "description": "An engaging 1-3 sentence description of your lab.",
    "headline": "Projects in Complex Trait Genetics",
    "publisher": {
      "@type": "Organization",
      "logo": { "@type": "ImageObject", "url": "/images/icon.png" }
    },
    "url": ""
  }
</script>

<link rel="alternate" type="application/rss+xml" href="/feed.xml">

  <!-- Google Fonts -->
<!-- automatically get url from fonts used in theme file -->

<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?display=swap&&family=Barlow:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600&amp;family=Roboto+Mono:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600" rel="stylesheet">

<!-- Font Awesome icons (load asynchronously due to size) -->

<link href="https://use.fontawesome.com/releases/v6.3.0/css/all.css" rel="preload" as="style" onload="this.onload = null; this.rel = 'stylesheet';">
<noscript>
  <link href="https://use.fontawesome.com/releases/v6.3.0/css/all.css" rel="stylesheet">
</noscript>

  <!-- third party styles -->
<!-- https://stylishthemes.github.io/Syntax-Themes/pygments/ -->
<link href="https://cdn.jsdelivr.net/gh/StylishThemes/Syntax-Themes/pygments/css-github/pygments-tomorrow-night-eighties.css" rel="stylesheet">

<!-- include all sass in styles folder -->


  
    <link href="/_styles/-theme.css" rel="stylesheet">
  

  
    <link href="/_styles/alert.css" rel="stylesheet">
  

  
    <link href="/_styles/all.css" rel="stylesheet">
  

  
    <link href="/_styles/anchor.css" rel="stylesheet">
  

  
    <link href="/_styles/background.css" rel="stylesheet">
  

  
    <link href="/_styles/body.css" rel="stylesheet">
  

  
    <link href="/_styles/bold.css" rel="stylesheet">
  

  
    <link href="/_styles/button.css" rel="stylesheet">
  

  
    <link href="/_styles/card.css" rel="stylesheet">
  

  
    <link href="/_styles/checkbox.css" rel="stylesheet">
  

  
    <link href="/_styles/citation.css" rel="stylesheet">
  

  
    <link href="/_styles/code.css" rel="stylesheet">
  

  
    <link href="/_styles/cols.css" rel="stylesheet">
  

  
    <link href="/_styles/dark-toggle.css" rel="stylesheet">
  

  
    <link href="/_styles/feature.css" rel="stylesheet">
  

  
    <link href="/_styles/figure.css" rel="stylesheet">
  

  
    <link href="/_styles/float.css" rel="stylesheet">
  

  
    <link href="/_styles/font.css" rel="stylesheet">
  

  
    <link href="/_styles/footer.css" rel="stylesheet">
  

  
    <link href="/_styles/form.css" rel="stylesheet">
  

  
    <link href="/_styles/grid.css" rel="stylesheet">
  

  
    <link href="/_styles/header.css" rel="stylesheet">
  

  
    <link href="/_styles/heading.css" rel="stylesheet">
  

  
    <link href="/_styles/highlight.css" rel="stylesheet">
  

  
    <link href="/_styles/icon.css" rel="stylesheet">
  

  
    <link href="/_styles/image.css" rel="stylesheet">
  

  
    <link href="/_styles/link.css" rel="stylesheet">
  

  
    <link href="/_styles/list.css" rel="stylesheet">
  

  
    <link href="/_styles/main.css" rel="stylesheet">
  

  
    <link href="/_styles/paragraph.css" rel="stylesheet">
  

  
    <link href="/_styles/portrait.css" rel="stylesheet">
  

  
    <link href="/_styles/post-excerpt.css" rel="stylesheet">
  

  
    <link href="/_styles/post-info.css" rel="stylesheet">
  

  
    <link href="/_styles/post-nav.css" rel="stylesheet">
  

  
    <link href="/_styles/quote.css" rel="stylesheet">
  

  
    <link href="/_styles/rule.css" rel="stylesheet">
  

  
    <link href="/_styles/search-box.css" rel="stylesheet">
  

  
    <link href="/_styles/search-info.css" rel="stylesheet">
  

  
    <link href="/_styles/section.css" rel="stylesheet">
  

  
    <link href="/_styles/table.css" rel="stylesheet">
  

  
    <link href="/_styles/tags.css" rel="stylesheet">
  

  
    <link href="/_styles/textbox.css" rel="stylesheet">
  

  
    <link href="/_styles/tooltip.css" rel="stylesheet">
  

  
    <link href="/_styles/util.css" rel="stylesheet">
  


<!-- include all css in styles folder -->



  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2" defer></script>
<script src="https://unpkg.com/tippy.js@6" defer></script>
<script src="https://unpkg.com/mark.js@8" defer></script>

<!-- include all js in scripts folder -->


  <script src="/_scripts/anchors.js"></script>

  <script src="/_scripts/dark-mode.js"></script>

  <script src="/_scripts/fetch-tags.js"></script>

  <script src="/_scripts/search.js"></script>

  <script src="/_scripts/site-search.js"></script>

  <script src="/_scripts/tooltip.js"></script>


</head>

  <body>
    







<header class="background" style="--image: url('/images/background.jpg')" data-dark="true">
  <a href="/" class="home">
    
      <span class="logo">
        
          <svg xmlns="http://www.w3.org/2000/svg" viewbox="-40 -60 80 100">
  <style>
    .bubble {
      animation: float 2s ease-out both infinite var(--delay);
    }
    @keyframes float {
      0% {
        opacity: 0;
      }
      50% {
        transform: translateY(0);
        opacity: 0;
      }
      75% {
        opacity: 1;
      }
      100% {
        opacity: 0;
        transform: translateY(-40px);
      }
    }
  </style>
  <g fill="currentColor" opacity="0.5">
    <circle class="bubble" cx="0" cy="-10" r="3" style="--delay: 0.1s"></circle>
    <circle class="bubble" cx="0" cy="-10" r="3" style="--delay: 0.4s"></circle>
    <circle class="bubble" cx="0" cy="-10" r="3" style="--delay: 1.1s"></circle>
  </g>
  <path fill="#38bdf8" d="
      M 0 -22.5
      L -19.5 -11.25
      L -19.5 11.25
      L 0 22.5
      L 19.5 11.25
      L 19.5 -11.25
      z
    "></path>
  <path fill="#bae6fd" d="
      M 0 -22.5
      L -19.5 -11.25
      L 0 0
      L 19.5 -11.25
      z
    "></path>
  <path fill="none" stroke="currentColor" stroke-width="5" d="
      M -18 -53
      L -10 -53
      L -10 -29.2
      L -30.3 -17.5
      L -30.3 17.5
      L 0 35
      L 30.3 17.5
      L 30.3 -17.5
      L 10 -29.2
      L 10 -53
      L 18 -53
    "></path>
</svg>

        
      </span>
    
    
      <span class="title" data-tooltip="Home">
        
          <span>Voight Lab</span>
        
        
      </span>
    
  </a>

  <input class="nav-toggle" type="checkbox" aria-label="show/hide nav">

  <nav>
    
    
      
        <a href="/research/" data-tooltip="Research Interests and Scientific Activities">
          Research
        </a>
      
    
      
        <a href="/team/" data-tooltip="About Our Team">
          Team
        </a>
      
    
      
        <a href="/tools/" data-tooltip="Published Methods">
          Tools
        </a>
      
    
      
        <a href="/data/" data-tooltip="Datasets">
          Data
        </a>
      
    
      
        <a href="/publications/" data-tooltip="Published Works">
          Publications
        </a>
      
    
      
        <a href="/join/" data-tooltip="Join our Team!">
          Join Us!
        </a>
      
    
      
        <a href="/calendar/" data-tooltip="Ben Voight's Public Calendar">
          Calendar
        </a>
      
    
      
        <a href="/contact/" data-tooltip="Email, address, and geolocation">
          Contact
        </a>
      
    
  </nav>
</header>

    <main>
      <!--
  modify main content of page:
  - add section breaks
  - attach section properties
  - wrap each table in div to allow for scrolling
  - filter out blank sections
-->








  
  
  

  <section class="background" data-size="page">
    <h1 class="center">Projects in Complex Trait Genetics</h1>

<div class="post-info">
  
    
    
      <span data-tooltip="Author">
        <i class="icon fa-solid fa-feather-pointed"></i>
        <span>Ben Voight</span>
      </span>
    
  

  
  

  
    <span data-tooltip="Originally published on">
      <i class="icon fa-regular fa-calendar"></i>
      <span>January 11, 2024</span>
    </span>
  

  
    <span data-tooltip="Last updated on">
      <i class="icon fa-solid fa-clock-rotate-left"></i>
      <span>October 14, 2024</span>
    </span>
  
</div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->



<div class="button-wrapper">
    <a class="button" href="/publications" data-style="" aria-label="button">
      
      
        <span>Links to Pubs:</span>
      
    </a>
  </div>

<div class="tags" data-link="/publications">
    
      <a href="/publications?search=%22tag:%20Complex%20Traits%22" class="tag" data-tooltip='Show items with the tag "Complex Traits"'>
        Complex Traits
      </a>
    
      <a href="/publications?search=%22tag:%20T2D%22" class="tag" data-tooltip='Show items with the tag "T2D"'>
        T2D
      </a>
    
      <a href="/publications?search=%22tag:%20MASLD%22" class="tag" data-tooltip='Show items with the tag "MASLD"'>
        MASLD
      </a>
    
      <a href="/publications?search=%22tag:%20CAD/CVD%22" class="tag" data-tooltip='Show items with the tag "CAD/CVD"'>
        CAD/CVD
      </a>
    
      <a href="/publications?search=%22tag:%20QTL%20Mapping%22" class="tag" data-tooltip='Show items with the tag "QTL Mapping"'>
        QTL Mapping
      </a>
    
  </div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Complex-Traits#T2D" data-style="" aria-label="button">
      
      
        <span>T2D</span>
      
    </a>
  </div>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Complex-Traits#MASLD" data-style="" aria-label="button">
      
      
        <span>MASLD</span>
      
    </a>
  </div>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Complex-Traits#QTL-Mapping" data-style="" aria-label="button">
      
      
        <span>QTL Mapping</span>
      
    </a>
  </div>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Complex-Traits#Collabs" data-style="" aria-label="button">
      
      
        <span>Additional Collaborations</span>
      
    </a>
  </div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->

<p><a id="T2D"></a></p>

<h2 id="discovery-and-analysis-type-2-diabetes-and-its-complications">Discovery and Analysis: Type-2 Diabetes and its Complications</h2>
<p>I have a longstanding interest in dissection of the genetic basis of type 2 diabetes (T2D). A great deal of progress has been made in characterizing the basis of the disease, in no small part to the advent of genome-wide association studies (GWAS) and the scale to which data can be collected. My efforts over my career thusfar has contributed to data collection and analysis that has resulted in the discovery of hundreds of associated loci to date. While work continues to expand discovery of new loci for the disease, expanding especially towards characterization of genetic liability across diverse cohorts, a great deal of work still remains “post” genome-wide association study. My fundamental drive is to identify genes and pathways that are therapeutically actionable, to expand the collection of treatments available, perhaps personally tailored to the sub-structure of genetic risk to T2D that an individual might carry. I believe genomics data collected in humans is central to that effort: identifying relevant leads, the therapeutic direction of effect, the ‘horsepower’ of specific human genetic targets, and so forth. As such, there are a range of GWAS and post-GWAS activities that are of active interest in the lab, with project invariably collaborative and large-scale with the <a href="Consortia#T2D-GGI">T2D-GGI</a> and <a href="Consortia#MVP">MVP</a> consortia. To enumerate a couple of areas interest:</p>

<ol>
  <li>
    <p><strong>Locus Discovery</strong>. Our most recent work has reported the assembly &gt;2.5M participants for the largest multi-ancestry GWAS of T2D to date, which lead to 145 newly loci for disease and bring the number of distinct loci to 611 and signals to 1289. However, this still did not bring the complete collection of data to bear. Ongoing data collection continues with AllofUs, MVP, and potentially bringing in cohorts to cover population groups that were not as well represented in this new effort as would have been desired (e.g., from South Asia).</p>
  </li>
  <li>
    <p><strong>Signal Fine Mapping</strong>. Mapping causal variants is directly relevant to causal gene identification, understanding mechanisms, and disease progression. Signal fine mapping is the appropriate statistical method to identify multiple, statistically independent signals associated, and is important as it: (i) extends the heritability explained by genetics, allowing us to deduce the relevance of causal variants shared across – but also potentially unique to – ancestry groups, (ii) allows us to connect association signals to additional metabolic and progression traits directly, and (iii) provides additional leads to connect to a putative causal nearby gene.
In our initial efforts for T2D-GGI, we applied fairly coarse (heuristic, but not quantitative) approach to capture putatively independent signals of association at each distinct T2D-associated locus. However, the toolkit of available approaches and data sets to deeply characterize the full spectrum of allelic heterogeneity comprehensively have never been better. I am particularly interested in application of these tools for T2D-GGI, to bring together true ‘in sample’ LD reference panels for large biobanks with summary data they contributed to meta-analysis to attack this problem.</p>
  </li>
  <li>
    <p><strong>Causal inference</strong>. Within the T2D-GGI, collaborations are ongoing to use the framework of Mendelian randomization (MR), QTL data, and multi-ancestry information to triangulate on variants and their predicted effects on nearby genes, to identify direction of effects and ‘horsepower’ of given perturbations of that gene to risk to T2D that might be expected. In addition, I am also interested in understanding non-linear effects of established causal risk factors and lifestyle factors in the context of T2D genetic liability. These questions can be addressed with the framework of MR, but require individual level data - which is available in MVP. Use of these data potentially further facilitates exploration of the role of causal risk factors for T2D across diversity ancestries, and to relate those risk factors potentially to risk of complications of T2D.</p>
  </li>
  <li>
    <p><strong>Colocalizaton: Variant-to-Gene Mapping and multi-trait physiology</strong>. It is highly desirable to connect non-coding variants – the lion’s share of tagged association at T2D risk loci – with a cognate, putatively relevant gene <em>in cis</em>. This can be achieved computationally through varaint-to-gene mapping approaches, e.g. using variation associated with change in transcript abundance (eQTLs) in relevant tissues or cell types (e.g., beta-cells of pancreatic islets). We are interested in attacking the variant-to-gene mapping problem generally, through analysis of existing data as well as generating new QTL data sets for this purpose - but also in the context of credible sets identified from signal fine-mapping efforts. In addition, we are also interested in attaching the complete spectrum of physiologic associations at each signal of association. We are interested in this approach, to (i) help understand the traits that might be expected to be perturbed (which may be beneficial or antagonistic to therapeutic indications), (ii) as a vehicle to help characterize pleiotropy generally in the genome, and (iii) to potentially obtain leads for the potential tissue or cell type of action that may be physiologically implicated at the associated signal.</p>
  </li>
  <li>
    <p><strong>Genetic Sub-Typing via Trait Clustering</strong>. In our recent work, and consistent with reports from others, it is possible to group T2D signals by cardiometabolic and anthropometric trait associations at those loci. These clusters appear to map onto to distinctive functional genomic and tissue groupings, e.g., beta-cell biology vs. adipose / obesity biology. In our paper, we demonstrated initial evidence that these sub-types may also diffferent associate with risk of T2D complications, though clearly more work to fuel stratification are necessary. In additional collaborative work within the T2D-GGI, we are exploring and contrasting different approaches to clustering, and are interested in evaluating new approaches to clustering fueled by additional data or new ways to approach the problem (e.g., colocalization or fine-mapping). The objective is potentially to develop and identify types that predict downstream complications (e.g., CAD, Kidney disease, retinopathy, etc.), which could potentially be transformative if successful.</p>
  </li>
</ol>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Complex-Traits" data-style="" aria-label="button">
      
      
        <span>Back to Top</span>
      
    </a>
  </div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->

<p><a id="MASLD"></a></p>

<h2 id="discovery-and-analysis-metabolic-associated-steatotic-liver-disease-and-its-progression">Discovery and Analysis: Metabolic-associated steatotic liver disease and its Progression</h2>
<p>Through involvement with MVP, I have recently become interested in dissecting the genetic basis of metabolic-associated steatotic liver disease (MASLD, previously referred to as NAFLD) and its progression. Our efforts involve not just locus discovery for the trait (or proxy) and progression phenotypes, but also post-GWAS analyses, and is a collaborative activity with many investigators at Penn and the VA, including Kyong-Mi Chang and Marijana Vujkovic, among many others.</p>

<ol>
  <li>
    <p><strong>Locus Discovery</strong>. Following along <a href="https://www.nature.com/articles/s41588-022-01078-z">our previously work</a>, efforts continue to map new susceptibility loci for the disease, or for related traits like liver cancer or cirrhosis. As data sets continue to grow, we are intrested in aggregating them for novel discovery as well as for down-stream analyses.</p>
  </li>
  <li>
    <p><strong>Signal fine mapping</strong>. Mapping causal variants is directly relevant to causal gene identification, understanding mechanisms, and disease progression. Signal fine mapping is the appropriate statistical method to identify multiple, statistically independent signals associated with disease. These allow us to extend heritability explained as well as find new targets for variant-to-gene mapping. As with T2D, I am particularly interested in application of novel tools and applications in MVP to deeply characterize the full spectrum of allelic heterogeneity comprehensively have never been better. The collection of liver disease and progression traits are opportune target for these efforts.</p>
  </li>
  <li>
    <p><strong>Multi-Trait mapping</strong>. MASLD by new definition carries with it at least one cardiometabolic trait association. I am interested in characterizing the trait physiology perturbed at MASLD and progression traits, identifying loci that are associated with lean MASLD, and performing multi-trait GWAS to expand the catalog of relevant associations that could be targets for functional studies beyond our primary scan.</p>
  </li>
  <li>
    <p><strong>Functional validation of MASLD Loci in the Zebrafish model system</strong>. Of established MASLD associations, a few transcripts identified through GWAS have been assigned causality for relevant phenotypic traits, such as liver fat, injury, or fibrosis/cirrhosis. These transcripts harbor variants with positive or negative risk associations that act through loss- and gain-of-function mechanisms in either hepatocytes or communicating cell types (immune). But like other complex diseases, many of the leading variants associated with MASLD are non-coding, which do not demarcate a clear candidate gene. While computational approaches can narrow down the list of potential candidates to reduce the number of genes in the list, candidate genes nominated in this way still require functional validation to demonstrate a phenotypic effect, direction of effect, and underlying mechanisms. Because we have dozens of MASLD candidate genes to evaluate, we need <em>in vivo</em> model system that facilitates tractable throughput, is scalable, and yet retains phenotypic depth and relevance. As such, we are embarked to build a collaboration with den Hoed Lab (Uppsala) to use zebrafish as a model system for validation. Zebrafish have a well-annotated genome with orthologues of at least 71% of human genes, or 82% of human disease-associated genes. Our results so far show strong directional consistency in effects on liver fat across zebrafish larvae and humans or mice for genes previously implicated through pLOF variants.</p>
  </li>
</ol>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Complex-Traits" data-style="" aria-label="button">
      
      
        <span>Back to Top</span>
      
    </a>
  </div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->

<p><a id="QTL-Mapping"></a></p>

<h2 id="quantitative-trait-locus-mapping-activities">Quantitative Trait Locus Mapping Activities</h2>
<p>We have recently expanded our interest and expertise in performing genetic mapping of molecular phenotypes, i.e., quantitative trait locus (QTL) discovery. Previously, we have performed scans in cell models for transcript abundance (eQTLs) or proportion of alternative spliced isoforms (sQTLs), and collaborate with local investigators to study primary tissues, including kidney (with Katalin Susztak). More recently, we are involved in several projects which utilize primary tissue in bulk and at single cell resolution, as well as expanding to chromatin accessibility (caQTL) mapping. Active projects include:</p>

<ol>
  <li>
    <p><strong>Single-cell eQTL and caQTL mapping for islet-cell types</strong>. In collaboration with Gaulton Lab (UCSD) and Grant Lab (Penn/CHOP), led by shared graduate student (Mary Ann Weidekamp), we are using single-cell data generated by the Human Pancreas Analysis Program which is collecting multi-omic readouts of pancreatic tissue obtained from deceased donors. These data include scRNA-Seq but also scATAC-Seq, and more recently, multiome RNA/ATAC-seq readouts. Our initial focus has been on mapping eQTLs in alpha, beta, and delta-cell populations, as well as additional cell types also captured in this experiment (e.g. acinar cells).</p>
  </li>
  <li>
    <p><strong>QTL mapping of B/T cells in pediatric cohort</strong>. In collaboration with Romberg Lab (CHOP), we are engaged in eQTL mapping in matched immune cell types (T cell, B cell) obtained from tonsil tissues extracted from pediatric cohorts, under baseline and stimulated conditions. These data are particularly novel, as they characterize immune cell populations from within a tissue context (not circulating in Blood, i.e. PBMCs), and derived from a pediatric cohort (age &lt;18). Our initial efforts in &gt;100 subjects have mapped ~14,000 eGenes in basal and stimulated states, including &gt;1,700 not previously reported in either large scale data sets (i.e., GTEx or eQTLGen) nor from catalogs of eQTLs from immune cell type data obtainde from PBMCs (i.e., the DICE consortium).</p>
  </li>
  <li>
    <p><strong>caQTL mapping in Bulk Liver</strong>. Lead by a graduate student (Bradon Wenz) during his time working with Casey Brown, we are focused on the analysis of ~180 primary bulk liver samples that he profiled using chromatin accessibility mapping via ATAC-Seq. The result is a massive map of regions of accessible chromatin in bulk liver tissue as well as a catalog of caQTLs. We are working together to finalize the analyses for these data.</p>
  </li>
</ol>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Complex-Traits" data-style="" aria-label="button">
      
      
        <span>Back to Top</span>
      
    </a>
  </div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->

<p><a id="Collabs"></a></p>

<h2 id="additional-collaborative-projects">Additional Collaborative Projects</h2>
<p>We also have a number of ongoing projects that generally involve collaborations or shared graduate students with another lab with complementary expertise our lab:</p>

<ol>
  <li>
    <p><strong>Fine-mapping and functional validation in Bone-mineral Density</strong>. Genetic factors strongly influence bone mineral density (BMD) and osteoporosis risk over time. To date, genome-wide association studies (GWAS) have discovered over 1,100 independent BMD signals; however, the causal variant(s) and respective effector gene(s) for the majority of these loci remain unknown. In collaboration with the Grant Lab, we <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02207-9">previously published the results</a> of a 3D genomics-based ‘variant-to-gene’ mapping method for BMD loci based on ATAC-seq, RNA-seq, and chromatin confirmation capture data from hMSC-derived osteoblasts. Led by graduate student (Mitch Conery) co-advised by Struan and myself, we are applying this approach, which links putative causal variants in open chromatin to candidate effector genes via chromatin interactions with open promoters, to a dataset derived from the human fetal osteoblast 1.19 cell line (hFOB). For validation of both sets of findings at scale, we conducted a CRISPRi screen in hFOBs differentiated for five days. In addition, Mitch is performing multi-trait signal fine-mapping for these signals across a range of related traits, to inform further prioritization efforts and potentially link BMD signals to novel biological and physiological mechanisms.</p>
  </li>
  <li>
    <p><strong>Genome-wide, phenome-wide Association studies in MVP</strong>. I was fortunate to join colleagues nation wide, to report <a href="https://www.medrxiv.org/content/10.1101/2023.06.28.23291975v1">a genome-wide, Phenome-wide association study across MVP participants</a>. The scope and volume of computation require to complete this project require collaborations with the DOE and supercomputer access with modified, efficient implementations of analysis tools, leading to new collaborations and inference at scale that has hitherto not been reported. There, across &gt;2,000 traits in &gt;630K MVP participants, we reported &gt;38,000 independent variants associated with 1 or more traits, performed fine-mapping which resulted in 6,318 singals from 613 traits to single-variant resolution. Future work extends in this vein to involve X-Chromosome analysis and expanded fine-mapping using multi-ancestry aware tools like SuSiEx, meSuSiE, mJAM, or flashfm.</p>
  </li>
</ol>

<div class="button-wrapper">
    <a class="button" href="/2024/01/11/Projects-Complex-Traits" data-style="" aria-label="button">
      
      
        <span>Back to Top</span>
      
    </a>
  </div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->


<div class="post-nav">
  <span>
    
      <i class="icon fa-solid fa-angle-left"></i> Previous post<br>
      <a href="/2024/01/11/Projects-Aspirational.html">
        Aspirational Projects
      </a>
    
  </span>
  <span>
    
      Next post <i class="icon fa-solid fa-angle-right"></i><br>
      <a href="/2024/01/11/Projects-MR.html">
        Projects in Mendelian Randomization
      </a>
    
  </span>
</div>
  </section>


    </main>
    


<footer class="background" style="--image: url('/images/background.jpg')" data-dark="true" data-size="wide">
  <!--
    <div>
      Extra details like contact info or address
    </div>
  -->

  <div>
    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="mailto:bvoight28@gmail.com" data-tooltip="Email" data-style="bare" aria-label="Email">
      <i class="icon fa-solid fa-envelope"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://orcid.org/0000-0002-6205-9994" data-tooltip="ORCID" data-style="bare" aria-label="ORCID">
      <i class="icon fa-brands fa-orcid"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://scholar.google.com/citations?user=FexB61cAAAAJ" data-tooltip="Google Scholar" data-style="bare" aria-label="Google Scholar">
      <i class="icon fa-brands fa-google"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://github.com/bvoightlab" data-tooltip="GitHub" data-style="bare" aria-label="GitHub">
      <i class="icon fa-brands fa-github"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://twitter.com/bvoight28" data-tooltip="Twitter" data-style="bare" aria-label="Twitter">
      <i class="icon fa-brands fa-twitter"></i>
      
    </a>
  </div>


    
  </div>

  <div>
    © 2024
    Voight Lab
      |   Built with
    <a href="https://github.com/greenelab/lab-website-template">
      Lab Website Template
    </a>
  </div>

  <input type="checkbox" class="dark-toggle" data-tooltip="Dark mode" aria-label="toggle dark mode" oninput="onDarkToggleChange(event)">
</footer>

  </body>
</html>
