---
title: Consortium-Level Activties
author: Ben Voight
---
{% include button.html text="Links to Pubs:" link="publications" %}
{% include tags.html tags="PanKbase, T2D-GGI, T2D-AMP-CMD, MVP" link="publications" %}

{% include section.html %}


{% include button.html text="T2D-GGI" link="2024/01/11/Consortia#T2D-GGI" %}
{% include button.html text="MVP" link="2024/01/11/Consortia#MVP" %}
{% include button.html text="T2D-AMP-CMD" link="2024/01/11/Consortia#T2D-AMP-CMD" %}
{% include button.html text="PanKbase" link="2024/01/11/Consortia#PanKbase" %}

{% include section.html %}
<a id="T2D-GGI"></a>

## The Type 2 Diabetes Global Genomics Initiative (T2D-GGI)
I have a long-standing commitment to mapping genetic susceptibility loci for type 2 diabetes (T2D). I was part of the first groups to perform GWAS successfully -- our efforts there to bring together data initially to make statistically robust (and replicated) observations formed the basis of genome-wide meta-analyses that we currently enjoy conventionally today. For T2D, I was part of [DIAGRAM](https://diagram-consortium.org/), the first consortium for meta-analysis of T2D data. Today, with long-standing colleagues (Andrew Morris and Ele Zeggini), I co-lead the Type 2 Diabetes Global Genomics Initiative (T2D-GGI), which carries the torch of that previous effort. Our initial effort has been to [assemble >2.5M participants for the largest multi-ancestry GWAS of T2D to date](https://www.medrxiv.org/content/10.1101/2023.03.31.23287839v1). Ongoing efforts of that group include use of those data to connect to gene and physiology (through statistical colocalization), refinement and characterization of genetic subtypes of diabetes through clustering methods, causal inference studies via Mendelian randomization, or signal fine-mapping. We are a collaborative, inclusive group and projects that utilize data in this area can give trainees exposure to and experience with fantastically effective large-scale multi-institutional consortium work, which is increasingly typical in human genomics activities.

{% include button.html text="Back to Top" link="2024/01/11/Consortia" %}

{% include section.html %}
<a id="MVP"></a>

## The VA Million Veteran Program
The VA's Million Veteran Program (MVP) is a national research program looking at how genes, lifestyle, military experiences, and exposures affect health and wellness in Veterans. Since launching in 2011, 1 million Veterans have joined MVP. It is the largest research effort at VA to improve health care for Veterans and one of the largest research programs in the world studying genes and health. I have a without compensation (WOC) appointment at the local VA here in Philadelphia (CMC), connected to Cardiometabolic Trait and Disease projects funded projects (to Kyong-Mi Chang at Penn, and Phil Tsao, Stanford) as well as projects related to CVD/PAD (to Scott Damrauer). My primary contribution focused around genetic discovery and analysis for T2D, liver disease, and CVD. This resource is particularly exciting to me with numerous opportunities for projects and productive scholarship, as its highly diverse representation at large sample size in a single repository make it ideal for complex trait genetic and population genetic discovery, characterization of genetic architecture, but also for causal inference using Mendelian randomization approaches that also have the opportunity to work at the individual level. To give one example of the potential of this: I was fortunate to join colleagues nation wide, to report [a genome-wide, Phenome-wide association study across MVP participants](https://www.medrxiv.org/content/10.1101/2023.06.28.23291975v1). The scope and volume of computation require to complete this project require collaborations with the DOE and supercomputer access with modified, efficient implementations of analysis tools, leading to new collaborations and inference at scale that has hitherto not been reported.

{% include button.html text="Back to Top" link="2024/01/11/Consortia" %}

{% include section.html %}
<a id="T2D-AMP-CMD"></a>

## The Type 2 Diabetes - Accelerating Medicines Partnership - Common Metabolic Disease - Functional Genomics Project (T2D AMP CMD FGP)
Large-scale genome-wide association studies (GWAS) have lead to the rapid identification of hundreds of T2D-associated loci. However, the mechanism(s) through which most of these loci influence disease susceptibility remain poorly understood. An interdisciplinary team at Penn (including Pat Seale, Wenli Yang, Struan Grant, Klaus Kaestner, Dan Rader, and myself) brings together experts in population genetics, T2D GWAS, biostatistics, metabolic tissue biology, human cellular disease modeling and T2D pathophysiology to tackle this critical knowledge gap. In collaboration with our consortium partners, our goal is to provide the diabetes research community with a robust pipeline for mapping T2D GWAS variants to effector genes and target tissues, identify new genes and biological pathways that modulate susceptibility to T2D, and define gene regulatory networks relevant to T2D with the goal of uncovering therapeutic ‘entry
points’ for developing new treatments. My efforts in this consortium lead on genetic discovery and characterization, closely partnering with groups focused on prioritizing putative effector transcripts using QTL mapping data. We also collaborate locally with Penn investigators to help interpret genetics data and to priortize leads that merit functional follow-up studies.

{% include button.html text="Back to Top" link="2024/01/11/Consortia" %}

{% include section.html %}
<a id="PanKbase"></a>

## PanKbase: Human Islet Research Network (HIRN) Pancreas Knowledgebase Program (PanKbase)
A wealth of data has been generated in human pancreas donors in initiatives supported by the Human Islet Research Network (HIRN) which can be used to address open questions in type 1 diabetes, yet these data are currently both underutilized and in formats inaccessible to many researchers which prohibits insights. To address this gap, the NIDDK has assembled two groups of highly accomplished researchers to create a pancreatic knowledge base - *PanKbase* - which leverages expertise in computational biology and data science, type 1 diabetes, islet biology, knowledge base engineering, and engagement and outreach. This is an incredibly new collaboration (as of Feb 2024); I expect more details to emerge about the goals, objectives, and deliverables for the project as the group organized and gets situated.

{% include button.html text="Back to Top" link="2024/01/11/Consortia" %}